pkg/operator/apis/config/v1alpha1/types.go

```diff
@@ -63,6 +63,8 @@ type ControllerConfiguration struct {
 	NetworkPolicy NetworkPolicyControllerConfiguration `json:"networkPolicy"`
 	// VPAEvictionRequirements is the configuration for the VPAEvictionrequirements controller.
 	VPAEvictionRequirements VPAEvictionRequirementsControllerConfiguration `json:"vpaEvictionRequirements"`
+	// ExtensionVirtualCluster defines the configuration of the extension virtual cluster controller.
+	ExtensionVirtualCluster ExtensionVirtualClusterControllerConfiguration `json:"extensionVirtualClusterConfig"`
```

Comment:
```suggestion
	ExtensionVirtualCluster ExtensionVirtualClusterControllerConfiguration `json:"extensionVirtualCluster"`
```

Also, adapt

- https://github.com/gardener/gardener/blob/master/example/operator/10-componentconfig.yaml
- https://github.com/gardener/gardener/blob/a67bdce51a379c4040229cffe9eb3c35c938c3f7/charts/gardener/operator/templates/_helpers.tpl#L35-L118

Reply:
This comment does not seem to be addressed yet